clifford/__init__.py

```diff
@@ -53,6 +53,9 @@
 from numpy import linalg, array,zeros
 import numba
 
+from clifford.gafileio import write_ga_file, read_ga_file
```

Comment:
i would have just called this `io`

Reply:
Agreed, I think that is nicer!

Reply:
Will fix it with some other improvements to the tooling today :)